Testskript.R

```diff
@@ -1,11 +1,12 @@
 
-# That function looked way to easy, please at a more complicated one!
+# I want a dinosaur please! Preferably a tyrannosaurus
 
-# Hallo Jule, das ist unser Testskript :-)    
-# :P
+add_numbers <- function(Jule, Anne) {
+  Nicklas <- (Jule + Anne)
+  return(Nicklas)
+}
 
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 9)
 
-# Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
-# :P
-
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+Nicklas(Jule, Anne)
```

Comment:
Great work! Please use the new function name instead of `Nicklas`. The example won't work otherwise.

Reply:
Thanks for the reminder, I blame that mistake on being in a hurry at the end :)